packages/fxa-settings/src/components/ConfirmWithLink/index.tsx

```diff
@@ -0,0 +1,117 @@
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
+import { FtlMsg } from 'fxa-react/lib/utils';
+import { ReactComponent as Mail } from './graphic_mail.svg';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { useFtlMsgResolver } from '../../models/hooks';
+
+export type ConfirmWithLinkProps = {
+  confirmWithLinkPageStrings: ConfirmWithLinkPageStrings;
+  email: string;
+  // TODO : update type definition once callback function is defined
+  goBackCallback?: () => void;
+  isOpenWebmailButtonVisible?: boolean; // TO-DO: Replace broker functionality which gives us this value (provider?)
+  resendEmailCallback: () => void;
+};
+
+export type WebmailValues = {
+  buttonText: string;
+  link: string;
+};
+
+export type ConfirmWithLinkPageStrings = {
+  headingText: string;
+  headingFtlId: string;
+  instructionText: string;
+  instructionFtlId: string;
+};
+
+const ConfirmWithLink = ({
+  confirmWithLinkPageStrings,
+  email,
+  goBackCallback,
+  isOpenWebmailButtonVisible = false,
+  resendEmailCallback,
+}: ConfirmWithLinkProps) => {
+  const ftlMsgResolver = useFtlMsgResolver();
+
+  const localizedAriaLabel = ftlMsgResolver.getMsg(
+    'confirm-signup-aria-label',
+    'An enveloppe containing a link'
+  );
+
+  // TO-DO: Replace utility that gets these values by matching the email provider via regex.
+  const getWebmailValues = (email: string) => {
+    // TODO replace hardcoded email provider and link - create a utility for this, see open-webmail-mixin.js
+    const emailProvider = 'Gmail';
+    return {
+      emailProvider,
+      link: `https://mail.google.com/mail/u/?authuser=${email}`,
+    };
+  };
+
+  const webmailValues = getWebmailValues(email);
+
+  /*
+    TO-DO:
+        - Fix up the alert bar for any errors (such as from resending the email) or success.
+        - wire up the functionality to resend an email
+  */
+
+  return (
+    <>
+      <div className="mb-4">
+        <FtlMsg id={confirmWithLinkPageStrings.headingFtlId}>
+          <h1 className="card-header">
+            {confirmWithLinkPageStrings.headingText}
+          </h1>
+        </FtlMsg>
+      </div>
+      <section>
+        <div className="flex justify-center">
+          <Mail role="img" aria-label={localizedAriaLabel} />
+        </div>
+        <FtlMsg id={confirmWithLinkPageStrings.instructionFtlId}>
+          <p className="my-4 text-sm">
+            {confirmWithLinkPageStrings.instructionText}
+          </p>
+        </FtlMsg>
+        <div className="flex flex-col gap-3">
+          {isOpenWebmailButtonVisible && (
+            <div>
+              <FtlMsg id="confirm-with-link-webmail-link">
+                <LinkExternal
+                  href={webmailValues.link}
+                  className="link-blue text-sm"
+                >
+                  Open {webmailValues.emailProvider}
```

Comment:
I think all of these links need another class somewhere on the container, the clickable space extends the full width:

<img width="679" alt="image" src="https://user-images.githubusercontent.com/13018240/215851231-ce4f3b01-3c9e-4106-8a24-f071eb26d1f8.png">

Reply:
Good eye!